Task_9/GetItem.java

```diff
@@ -0,0 +1,68 @@
+package Task_9;
+
+import java.io.*;
+import java.nio.file.*;
+import java.nio.file.attribute.*;
+import java.text.SimpleDateFormat;
+
+/**
+ * Class consists of methods that get information about directory or file.
+ * @author Bogatko Irina
+ * @version 1.0
+ * @since 18.10.2016
+ */
+public class GetItem {

```

Comment:
Not a good name - what does it really get and what kind of item? Let be more specific - FileItemInformation or something like that